tasks/messages/message_test.go

```diff
@@ -122,20 +126,95 @@ func TestParseMessageParams(t *testing.T) {
 			params:      nil,
 			actorCode:   builtin3.AccountActorCodeID,
 			wantMethod:  "PubkeyAddress",
-			wantEncoded: `""`,
+			wantEncoded: ``,
 			wantErr:     false,
 		},
 		{
 			// Derived from message bafy2bzaceaoyvylhmpn6foboyajbbcjvczszyjs4do7mgp4vuutbgrk5z42fu
-			// Account actor methods may receive unexpect params
+			// Account actor methods may receive unexpect params, they should not be parsed as they will result in invalid json.
 			name:        "issue-772",
 			method:      2,
 			params:      mustDecodeBase64(t, "dHJhbnNmZXI="),
 			actorCode:   builtin3.AccountActorCodeID,
 			wantMethod:  "PubkeyAddress",
-			wantEncoded: `"dHJhbnNmZXI="`,
+			wantEncoded: ``,
 			wantErr:     false,
 		},
+		// Derived from message bafy2bzaceclkxipts274qqchuqn4ri6w5cce26f7lbxljfpbsgalv4zeq4huk
```

Comment:
@iand I have attempted to check that https://github.com/filecoin-project/lily/issues/741 via this test. Mind giving this a second look? I don't feel very certain on the fix 😶‍🌫️